DEV/Info.plist

```diff
@@ -61,6 +61,9 @@
 	<key>UISupportedInterfaceOrientations</key>
 	<array>
 		<string>UIInterfaceOrientationPortrait</string>
+		<string>UIInterfaceOrientationPortraitUpsideDown</string>
+		<string>UIInterfaceOrientationLandscapeLeft</string>
+		<string>UIInterfaceOrientationLandscapeRight</string>
```

Comment:
The landscape options were purposely removed could you remove them from your PR, please?

Reply:
Doesn't' seem to hurt anything with the display and the way desk is setup it helps me leave the iPad connected to my Mac while debugging

Reply:
There is a bug around landscape (#25) and please see #27. However, you are more than welcome to keep these changes local.